src/hotspot/share/nmt/nmtPreInit.hpp

```diff
@@ -23,19 +23,19 @@
  *
  */
 
-#ifndef SHARE_SERVICES_NMT_PREINIT_HPP
-#define SHARE_SERVICES_NMT_PREINIT_HPP
+#ifndef SHARE_NMT_NMT_PREINIT_HPP
+#define SHARE_NMT_NMT_PREINIT_HPP
 
 #include "memory/allStatic.hpp"
-#ifdef ASSERT
-#include "runtime/atomic.hpp"
-#endif
-#include "services/memTracker.hpp"
-#include "utilities/checkedCast.hpp"
+#include "nmt/memTracker.hpp"
 #include "utilities/debug.hpp"
 #include "utilities/globalDefinitions.hpp"
 #include "utilities/macros.hpp"
 
+#ifdef ASSERT
```

Comment:
The blank line at 34 is not following the style for our conditional includes. Remove it, or better yet skip conventionalize the include of runtime/atomic.hpp since it just adds to noise to the file.

Reply:
Removed the blank line, I think the noise is meaningful in this case.